### ISSUE TITLE
Override DB values with JSON

### ISSUE THREAD

**Issue by ataylorme** [forumone/wp-cfm#98]
Related to #95. Expose filters so bundles can be set as the source of truth, rather than the database, for all tracked options.

Example use cases: 

* force transactional test mode in non-production
* ensure dev plugins are disabled in production

**Comment by elvismdev** [comment]
@ataylorme PR #105 is good to test for this feature. 

This introduces a new filter `wpcfm_is_ssot` that returns a boolean value (by default it returns `false`). Making the filter return `true` will skip loading from DB tracked plugin options on WP-CFM. Then instead it will read them directly from the bundle's config files.

It must be added via a MU Plugin.

```php
// wp-content/mu-plugins/wpcfm-config.php

/**
 * @return bool
 */
add_filter( 'wpcfm_is_ssot', '__return_true' );
```

Note that in that PR there is included also the latest work for issue #95 .

**Comment by elvismdev** [comment]
Feature now merged into `master`
